src/App.js

```diff
@@ -151,9 +163,7 @@ function App() {
           <AdminAuthPageRoute path="/judging/admin">
             <JudgingAdmin />
           </AdminAuthPageRoute>
-          <AuthPageRoute path="/judging/view/:id">
-            {params => <JudgingView id={params.id} />}
-          </AuthPageRoute>
+          <Route path="/judging/view/:id" component={JudgingViewContainer} />
```

Comment:
because we need `params` to be accessible directly under the `<Route>` component